app/src/main/webapp/index.html

```diff
@@ -71,11 +71,11 @@
           });
         }
 
-        $(document).ready(() => {
-           $("#invoke").click((e) => {
+        $(document).ready(function () {
+           $("#invoke").click(function (e) {
             e.preventDefault();
               //invokeAPI();
-              const n = $("#name").val() || "World";
+              const name = $("#name").val() || "World";
```

Comment:
Can we avoid the `|| "World"` bit? Otherwise we're not really demoing the part where we don't send the `name` parameter.